.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+Please read [The Change Author’s Guide](https://google.github.io/eng-practices/review/developer/) before opening any pull requests.
+
+## Related Issue
+
+## Description
+
+## Motivation and Context
+
+## How Has This Been Tested?
```

Comment:
The tests will be run automatically and shows up here: 
<img width="1063" alt="Screen Shot 2020-06-20 at 11 26 15 AM" src="https://user-images.githubusercontent.com/3537801/85215537-07910200-b32f-11ea-829c-a92fa7c127de.png">

So, we don't need to mention how it's tested. It's automatic.